blog/themes/3mdeb/layouts/partials/widget-most-popular.html

```diff
@@ -0,0 +1,9 @@
+<div class="card">
+    <div class="card-content">
+        <h1 class="title is-5">Popular Posts</h1>
+        {{ range (where .Site.RegularPages "Section" "post") | shuffle | first 5 }}
```

Comment:
@artur-rs couldn't find a function that can count pages by view. I wonder If there is any other solution than use .Site.GoogleAnalytics
At the moment I used Shuffle to show 5 random posts as "Popular Posts"